### ISSUE TITLE
Strange messaging with PhantomJS & BrowserStack

### ISSUE THREAD

**Issue by stevenvachon** [bitovi/testee#112]
```
Mocha "test.html" on PhantomJS 2.1.1 / Mac OS X 0.0.0:
```
```
Mocha "test.html" on IE 9.0.0 / Windows 7 0.0.0:
```
(notice the os versions)

Doesn't happen with Firefox, though:
```
Mocha "test.html" on Firefox 53.0.0 / Mac OS X 10.12.0:
```

**Comment by daffl** [comment]
This is either an issue with the [useragent](https://www.npmjs.com/package/useragent) library or PhantomJS not being able to report the operating system version.

**Comment by andrejewski** [comment]
Investigating, this was supposedly fixed in [phantomjs@2.1.1](https://github.com/ariya/phantomjs/commit/c7d7c656b3924e08c5488a58165289fbb1a512bb). Closing as this is not an issue with Testee directly.
